portalias/PORTALIAS.md

```diff
@@ -0,0 +1,75 @@
+#Portalias
```

Comment:
This file is now very out-of-date, since we are no longer using a flag to determine the mode. Also, a design doc of this sort doesn't belong in this repo alongside the code; if anything it should be placed in the GitHub wiki. Please delete this file.